src/app/courses-page/course/course-host-testing.component.spec.ts

```diff
@@ -0,0 +1,92 @@
+    title: 'Video Course 1. Name tag',
+    creationDate: new Date('2020, 08, 28'),
+    duration: {
+      hours: 1,
+      minutes: 38
+    },
+    description:
+      'Learn about where you can find course descriptions, what information they include, how they work, and details about various components of a course description. Course descriptions report information about a university or college\'s classes. They\'re published both in course catalogs that outline degree requirements and in course schedules that contain descriptions for all courses offered during a particular semester.'
+  };
+  deletedCourse: Course;
+  editedCourse: Course;
+
+  onDeleteCourse(course: Course): void {
+    this.deletedCourse = course;
+  }
+
+  onEditCourse(course: Course): void {
+    this.editedCourse = course;
+  }
+}
+
+describe('CourseComponent', () => {
+  let testHost: TestHostComponent;
+  let fixture: ComponentFixture<TestHostComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [CourseComponent, TestHostComponent]
+    });
+    fixture = TestBed.createComponent(TestHostComponent);
+    testHost = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should show correct title', () => {
+    const courseTitle = fixture.debugElement.query(By.css('.title'));
+    const courseTitleEl = courseTitle.nativeElement;
+    expect(courseTitleEl.textContent).toContain(testHost.course.title);
+  });
+
+  it('should show correct time', () => {
+    const courseTime = fixture.debugElement.query(By.css('.time'));
+    const courseTimeEl = courseTime.nativeElement;
+    expect(courseTimeEl.textContent).toContain(testHost.course.duration.hours);
+    expect(courseTimeEl.textContent).toContain(testHost.course.duration.minutes);
+  });
+
+  it('should show correct date', () => {
+    const courseDate = fixture.debugElement.query(By.css('.date'));
+    const courseTitleEl = courseDate.nativeElement;
+    const expectedMonth = testHost.course.creationDate.getMonth() + 1;
+    const expectedDay = testHost.course.creationDate.getDate();
+    const expestedYear = testHost.course.creationDate.getFullYear().toString().substr(-2);
+    const courseTitlePipedEl = `${expectedMonth}/${expectedDay}/${expestedYear}`;
+    expect(courseTitleEl.textContent).toContain(courseTitlePipedEl);
+  });
+
+  it('should show correct description', () => {
+    const courseDescription = fixture.debugElement.query(By.css('.information'));
+    const courseDescriptionEl = courseDescription.nativeElement;
+    expect(courseDescriptionEl.textContent).toContain(testHost.course.description);
+  });
+
+  it('should raise edit event when clicked (triggerEventHandler)', () => {
+    const editBtn = fixture.debugElement.query(By.css('.edit-btn'));
+
+    editBtn.triggerEventHandler('click', null);
+    expect(testHost.editedCourse).toBe(testHost.course);
+  });
+
+  it('should raise delete event when clicked (triggerEventHandler)', () => {
```

Comment:
I would name it `should raise delete item`, since you actually test here if the deleted component is raised to the host component upon click.
Same about other `its` description. It should mirror your expect clause in some way.

Reply:
Names are changed.